### PR TITLE
[dev-v2.7] Add 1.27 for k3s and rke2

### PR DESCRIPTION
**Pull request by rancher-max**
https://github.com/rancher/rancher/issues/44209

This also includes Jan 1.26 patch.

### CODE REVIEW

File: channels-rke2.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ appDefaults:
         defaultVersion: '1.24.x'
       - appVersion: '>= 2.7.2-0 < 2.7.5-0'
         defaultVersion: '1.25.x'
-      - appVersion: '>= 2.7.5-0 < 2.7.100-0'
+      - appVersion: '>= 2.7.5-0 < 2.7.11-0'
```
Comment (snasovich) [inline channels-rke2.yaml:15]: 👍

File: channels-rke2.yaml [hunk 2]
```diff
@@ -1635,7 +1652,7 @@ releases:
         version: 1.24.004
     featureVersions: *featureVersions-v1
   - version: v1.27.7+rke2r2
-    minChannelServerVersion: v2.7.11
+    minChannelServerVersion: v2.7.11-alpha1
```
Comment (snasovich) [inline channels-rke2.yaml:1655]: 👍

**Review by snasovich — APPROVED** [review]
LGTM, thank you for correcting some of the version ranges that were missed in previous changes that introduced these versions.

**Review by briandowns — APPROVED** [review]

**Review by tashima42 — APPROVED** [review]

**Review by brooksn — APPROVED** [review]

**Comment by VestigeJ** [comment]
Screenshot 2024-02-12 at 11 14 37 AM

**Pull request merged**